apps/ejabberd/src/ejabberd_listener.erl

```diff
@@ -173,13 +174,14 @@ listen_tcp(PortIPProto, Module, SockOpts, Port, IPS) ->
             ListenSocket;
         _ ->
             Res = gen_tcp:listen(Port, [binary,
+                                        {backlog, 100},
                                         {packet, 0},
                                         {active, false},
                                         {reuseaddr, true},
                                         {nodelay, true},
                                         {send_timeout, ?TCP_SEND_TIMEOUT},
                                         {keepalive, true},
-                                        {send_timeout_close, true}]),
+                                        {send_timeout_close, true} | SockOpts]),
```

Comment:
I see -- so that's very non-obvious, and someone may come along and change it later. Let's do it the right way so no "alternative" interpretation is possible.